jenkins/job-configs/kubernetes-jenkins/kubernetes-soak.yaml

```diff
@@ -1,15 +1,31 @@
-- job-template:
-    name: 'kubernetes-soak-weekly-deploy-{suffix}'
-    description: '{deploy-description} Test owner: {test-owner}'
+- soak_defaults: &soak_defaults
+    name: soak_defaults
+    branch: 'master'
     node: 'master'
+    provider-env: |
+        export KUBERNETES_PROVIDER="gce"
+        export E2E_MIN_STARTUP_PODS="1"
+        export KUBE_GCE_ZONE="us-central1-f"
+        export FAIL_ON_GCP_RESOURCE_LEAK="true"
+        export CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS="1"
     builders:
         - shell: |
             {provider-env}
```

Comment:
Since this is the only place that uses it, should we just move the export list to here?